README.md

```diff
@@ -173,24 +175,60 @@ For convenience, the bot will look for an open ART pull request and cherry-pick
 
 ## Lifecycle hooks
 
-User created scripts can be set up to be executed at specific points of the bot's lifecycle. 
-Specify a path to the script you want to run using the following parameters. These scripts can be any executable file.
+User-provided scripts can be configured to execute at specific points of the bot's lifecycle. 
+You can specify a path to the script you want to run using the following parameters. These scripts can be any executable file. The same lifecycle hook arguments can be specified multiple times to attach multiple scripts to the same hook. If multiple scripts are attached to the same lifecycle hook, they will execute in the order they are provided.
 
 - **`--pre-rebase-hook`**: Executed when repository is setup with `rebase` branch checked out on `source/branch` before rebase.
 - **`--pre-carry-commit-hook`**: Executed before carrying each commit during the rebase process. The upstream is merged into the rebase branch.
 - **`--post-rebase-hook`**: Executed after the rebase process is completed.
 - **`--pre-push-rebase-branch-hook`**: Executed before pushing the rebase branch to the remote repository.
 - **`--pre-create-pr-hook`**: Executed before creating the pull request.
 
-### Scripts stored inside the repository
+### Script sources
+
+Scripts can be loaded from local filesystem, from one of the remotes, or from the builtin scripts directory.
+
+#### Local file scripts
+
+Local file scripts can be specified either by their absolute path or by their relative path to the current working directory.
+
+##### Example
+
+```sh
+rebasebot ... \
+    --pre-rebase-hook /home/user/script.sh \
+    --pre-rebase-hook script.sh
+```
+
+#### Scripts stored inside the repository
 
 To ensure scripts stored within the repository are available in all stages of the rebase process specify their path as `git:gitRef:repo/relative/path/to/script`. This approach makes the scripts accessible throughout the rebase.
 
 *Note: `gitRef` can be a branch name, tag name, or commit hash.*
 
+##### Example
+    
+The following example will attach script to be run after rebase from `rebasebot/generate-script.sh` file stored on the `dest/main` branch.
+
+```sh
+rebasebot --post-rebase-hook git:dest/main:rebasebot/generate-script.sh
+```
+
+#### Builtin lifecycle hook scripts
+
+Some scripts are included in the bot repository itself. They are stored in the `rebasebot/builtin-hooks` directory.
+
+Builtin scripts are available via the `[BUILTIN]/` path prefix.
```

Comment:
The square brackets need escaping in a POSIX shell. Could we use a different prefix that does not need escaping, like for example `_BUILTIN_/`

Reply:
Good catch. Changed as suggested.